xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1030 @@
+    key: /certs/domain.key
+health:
+  storagedriver:
+    enabled: true
+    interval: 10s
+threshold: 3
+         </screen>
+     </step>
+     <step>
+      <para>
+       Now start the registry service and enable it at boot time:
+      </para>
+<screen>&prompt.root;<command>systemctl enable --now registry.service</command>
+         </screen>
+     </step>
+    </procedure>
+
+    <para>
+     Now you should have the registries set up and listening on port
+     <literal>5000</literal> on their respective servers.
+    </para>
+    <para>
+     You must now perform the shared update procedure to populate the registry.
+     Refer to: <xref linkend="sec.deploy.scenarios.airgap.update"/>.
+    </para>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.container-registry.clients">
+    <title>Client Configuration</title>
+    <para>
+     Using the airgapped mirror works exactly like using a traditional
+     on-premise mirror. You must configure the internal registry mirror in
+     &dashboard;.
+    </para>
+    <procedure>
+     <title>Configuring Registry Mirror</title>
+     <step>
+      <para>
+       Log in to &dashboard; on the &productname; admin node.
+      </para>
+     </step>
+     <step>
+      <para>
+       Navigate to <guimenu>Settings &rarr; Mirrors</guimenu> and create a
+       definition for your internal registry mirror as described in
+       <xref linkend="sec.admin.velum.mirror"/>.
+      </para>
+     </step>
+    </procedure>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.helm-charts">
+   <title>Helm Chart Repository Mirror</title>
+   <para>
+    The helm charts will require images available from a registry mirror. The
+    following steps will aid you in mirroring all images required by the &suse;
+    provided helm charts. To make use of the helm charts, you must complete
+    <xref linkend="sec.deploy.scenarios.airgap.container-registry"/>.
+   </para>
+   <sect3>
+    <title>Mirror Configuration</title>
+    <para>
+     You must have a working container image registry mirror in place inside
+     the air gapped network. This can be any webserver capable of hosting static
+     files. We recommend providing the webserver on your internal mirror server.
+    </para>
+    <procedure>
+     <title>Setting Up Chart Repository Mirror</title>
+     <step>
+      <para>
+       Install and configure a webserver (for example <link xlink:href="https://www.suse.com/documentation/sles-12/book_sle_admin/data/cha_apache2.html">Apache</link>)
+       on your airgapped mirror server.
+      </para>
+     </step>
+     <step>
+      <para>
+       SSH into your mirror server on the <literal>external</literal> network.
+      </para>
+     </step>
+     <step>
+      <para>
+       Download all helm charts from the helm chart repository using
+       <command>helm-mirror</command>.
+      </para>
+<screen>&prompt.user;helm-mirror https://kubernetes-charts.suse.com /chartserver/charts
```

Comment:
Wrong url, you have to use the SUSE helm chart. That's our only focus right now

Reply:
Not sure what's wrong with the url (what is the SUSE helm chart?) but this URL is also referenced below (procedure 2.7)